Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -1231,7 +1231,15 @@ public virtual float BaseFontSize
         {
             get => _baseFont ?? new("Segoe UI", BaseFontSize, FontStyle.Regular);
```

Comment:
This means that the return is never null.
Therefore remove the nullable return type

Reply:
Done